cmake/sysbuild/suit.cmake

```diff
@@ -368,6 +368,34 @@ function(suit_create_package)
     endif()
   endif()
 
+  if(SB_CONFIG_SUIT_MULTI_IMAGE_PACKAGE_BUILD)
+    include(${ZEPHYR_BASE}/../nrf/cmake/fw_zip.cmake)
+    include(${ZEPHYR_BASE}/../nrf/cmake/dfu_multi_image.cmake)
+
+    set(suit_multi_image_ids)
+    set(suit_multi_image_paths)
+    set(suit_multi_image_targets)
+
+    list(APPEND suit_multi_image_ids 0)
+    # Include the suit Envelope to the multi image package to store it in dfu_partition
+    list(APPEND suit_multi_image_paths "${SUIT_ROOT_DIRECTORY}root.suit")
+    list(APPEND suit_multi_image_targets "${SUIT_ROOT_DIRECTORY}root.suit")
+    # Include cache partition to the multi image package to store it in cache_partition
+    foreach(CACHE_PARTITION_NUM ${DFU_CACHE_PARTITIONS_USED})
```

Comment:
seems to be a common pattern of using upper case for local variables.

Please shift to lower case for varieable which are purely intended for internal use in this CMake file / functions.
Comment apply to all occurrences in this PR

Reply:
Can I do it in the next PR as a follow-up? I would like to merge it to unblock other work.